### ISSUE TITLE
Flood-Limits beachten

### ISSUE THREAD

**Issue by eknoes** [eknoes/covidbot#37]
Siehe https://github.com/python-telegram-bot/python-telegram-bot/wiki/Avoiding-flood-limits

Müssen wir dringend implementieren, damit unser Bot nicht versehentlich gesperrt wird.

**Comment by errotu** [comment]
Ist es nur für tägliche Berichte gefixt oder auch für den Fall, dass wir manuell eine Nachricht versenden (bspw. eine Korrektur)?

**Comment by eknoes** [comment]
Nein, dafür nicht.

**Comment by eknoes** [comment]
Nein, nur für die täglichen Berichte. Für den Rest ist es nicht so trivial, wir müssen also noch was implementieren - allerdings ist das nicht ganz so dringend!

**Comment by errotu** [comment]
Hat sich das mit den letzten Signal-Fixes erledigt oder siehst du noch Handlungsbedarf, @eknoes?

**Comment by eknoes** [comment]
Wenn unsere Nutzer*innenzahlen steigen wäre das theoretisch noch ein Problem, aber ich glaube der Uberspace wäre dann zu langsam. Aber falls wir mal auf einen performanteren Server wechseln müssten oder so definitiv relevant.

**Comment by eknoes** [comment]
Ist ein Problem, heute gab es bspw. einen Fehler bei Telegram: "telegram.error.RetryAfter: Flood control exceeded. Retry in 35.0 seconds"
